### ISSUE TITLE
Question about fold setting

### ISSUE THREAD

**Issue by zhi-xuan-chen** [MIC-DKFZ/nnUNet#2525]
Dear author,

I want to ask when the fold is set to all, is there no validation set? Does it use the entire training set for training?
